sympy/matrices/matrices.py

```diff
@@ -384,15 +384,56 @@ def cofactor_matrix(self, method="berkowitz"):
                          lambda i, j: self.cofactor(i, j, method))
 
     def det(self, method="bareiss", iszerofunc=None):
-        """Computes the determinant of a matrix.  If the matrix
-        is at most 3x3, a hard-coded formula is used.
-        Otherwise, the determinant using the method `method`.
+        """Computes the determinant of a matrix.
 
+        Parameters
+        ==========
+        method : string, optional
+            Specifies an algorithm to use for computing matrix determinant.
+
+            If the matrix is at most 3x3, the key is ignored and a hard-coded
+            formula is used. Otherwise, it will be defaulted with 'bareiss'.
```

Comment:
Possibly: `If the matrix is at most 3x3, a hard-coded formula is used and the specified method is ignored. Otherwise, it defaults to 'bareiss'.`